spec/global_id_utils_spec.rb

```diff
@@ -58,4 +58,66 @@
       GlobalID.find_many!(strs, ignore_missing: true)
     end
   end
+
+  describe 'GlobalID.scoped_find' do
+    let!(:model_class) do
+      module Fish
+        class NeonTetra < Base
+        end
+      end
+
+      ::Fish::NeonTetra
+    end
+
+    before { GlobalID.app = 'testapp' }
+    after { Fish.send(:remove_const, 'NeonTetra') if Fish.const_defined?('NeonTetra') }
+
+    context 'when called with scope objects' do
+      it 'evaluates multiple scopes' do
+        with_name_scope = ::GlobalIdUtils::Scope.new(:with_name, 'Nemo')
+        with_fins_scope = ::GlobalIdUtils::Scope.new(:with_fins)
+
+        expect(model_class).to receive(:with_name).with('Nemo').and_return(model_class)
```

Comment:
It's not a huge deal, but I would vote we actually define these methods/scopes. If we were to turn on rspec double verification like we use in portal this would blow up.

FWIW I think turning on double verification in this gem would probably be a good idea anyway, it's saved us from a few silly bugs in the past. It's just a couple lines in the spec_helper if you think it's worth it 
```ruby
RSpec.configure do |config|            
  config.mock_with :rspec do |mocks|   
    mocks.verify_partial_doubles = true
  end                                  
end                                    
```

Reply:
Good suggestion! This is done in https://github.com/wunderteam/globalid-utils/pull/8/commits/1eb3415821ff556d540c4f3aa1a02aea5cc6ec71